### PR TITLE
Adding a simple bufferedmutator based write performance test.

### PR DESCRIPTION
**Pull request by sduskis**

### CODE REVIEW

File: java/simple-performance-test/pom.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  <repositories>
+    <repository>
+      <id>snapshots-repo</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+      <releases><enabled>false</enabled></releases>
+      <snapshots><enabled>true</enabled></snapshots>
+    </repository>
+  </repositories>
+
+  <build>
```
Comment (lesv) [inline java/simple-performance-test/pom.xml]: PTAL @ the pom.xml in managed-vm-gae on how to do this so running local works.  (defaulting to Linux, but using Profiles for local -Pmac -Pwindows

File: java/simple-performance-test/pom.xml [hunk 2]
```diff
@@ -0,0 +1,95 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (lesv) [inline java/simple-performance-test/pom.xml]: We try to put a license file here.

**Comment by lesv** [comment]
Basically happy - just need two changes.

File: java/simple-performance-test/pom.xml (hunk 1 above)
Comment (sduskis) [inline java/simple-performance-test/pom.xml]: I really don't want to use the -P(mac|windows).  The concerns of managed-vm-gae are not a consideration for applications meant to run on a single machine.

Comment (lesv) [inline java/simple-performance-test/pom.xml]: I'm ok w/ your view, but I look at it that everything gets pushed to the cloud from a development as jars, not source.  It's too easy for a developer to push the wrong one.

Comment (lesv) [inline java/simple-performance-test/pom.xml]: What I'd like to see is a comment in the README -- I'll do that after you submit.

It still needs the license header - then LGTM

**Pull request merged**